gammapy/datasets/core.py

```diff
@@ -420,7 +420,12 @@ def read(cls, filename, filename_models=None, lazy=True, cache=True):
         return datasets
 
     def write(
-        self, filename, filename_models=None, overwrite=False, write_covariance=True
+        self,
```

Comment:
Can you profit of this PR to add the default value of the parameters in the docstring?

Reply:
done